src/components/pages/Home/Offerers/CreationLinks.jsx

```diff
@@ -0,0 +1,69 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { Link } from 'react-router-dom'
+
+import { isAPISireneAvailable } from 'store/selectors/data/featuresSelectors'
+import { UNAVAILABLE_ERROR_PAGE } from 'utils/routes'
+
+const CreationLink = ({ hasPhysicalVenue, hasVirtualOffers, offererId }) => {
+  const isVenueCreationAvailable = useSelector(isAPISireneAvailable)
+
+  const venueCreationUrl = isVenueCreationAvailable
+    ? `/structures/${offererId}/lieux/creation`
+    : UNAVAILABLE_ERROR_PAGE
+
+  const renderCreationLinks = insideCard => (
```

Comment:
```suggestion
  const renderCreationLinks = ({ insideCard }) => (
```
cf. suggestions l.50 et l.58